### ISSUE TITLE
400 Bad request

### ISSUE THREAD

**Issue by christophrumpel** [iboldurev/api-ai-php#2]
Hey @iboldurev ,

today it started given me this error:

```
GuzzleHttp\\Exception\\ClientException: Client error: `GET https://api.api.ai/v1/query?v=20150910&query=fsdfds` resulted in a `400 Bad Request
```

When I try to query a simple message like:

```
 $query = $this->apiClient->get('query', [
      'query' => $message,
]);
```

I checked the apiai token and my code and I'm not sure why I get this error. Can you help me?

Thx Christoph

**Comment by iboldurev** [comment]
What version the library you are using?
Today I updated to the new version of the library, is now sending queries occur through a post request

**Comment by iboldurev** [comment]
@christophrumpel I checked your code, it works.
Maybe you need to update your version of the library (composer update) to 0.2.3

**Comment by christophrumpel** [comment]
0.2.2, ah ok then I will try 0.2.3 👍 thx

**Comment by iboldurev** [comment]
@christophrumpel I am glad to help you, thank you for the use of the library.

**Comment by christophrumpel** [comment]
Thx, I am really glad you created it!

**Comment by christophrumpel** [comment]
Interesting, I got the same problem with another project where I use your package version 0.2.5.
I figured out, that I needed to add a session ID in order to make it work. Is this a bug? (I used the time() method for testing purpose)

```php
try {
    $client = new Client($this->accessToken);

    $query = $client->get('query', [
        'query'     => $message,
        'sessionId' => time()
    ]);

    $response = json_decode((string)$query->getBody(), true);
    dd($response);
} catch (\Exception $error) {
    dd($error);
}
```

**Comment by bottomoftheheap9000** [comment]
Christoph, how am I to resolve this issue? Im getting 400 error. What is the $this variable? My composer shows iboldurev/api-ai-php 0.2.5  API.ai php sdk

**Comment by bottomoftheheap9000** [comment]
How should my code look?

**Comment by bottomoftheheap9000** [comment]
https://gist.github.com/bottomoftheheap9000/2313449ce2fc5cfefc13c89136ed3151

**Comment by bottomoftheheap9000** [comment]
@christophrumpel @iboldurev

**Comment by christophrumpel** [comment]
Hey @bottomoftheheap9000 , in my example `$this->accessToken` refers to my variable. You can just put your access token in like `... new Client('your-access-token');`.

Have your tried to add a `sessionId` to the query params like in my example above? Because it is missing in your gist example.

**Comment by bottomoftheheap9000** [comment]
Hi! Thanks for responding! I've altered my code as advised, the page now loads (so it must be connecting to the api!), however Im not able to display the json response. Im not sure if Im receiving anything from api.ai, I have tried 

echo $response;
echo (string)$response['result']['fulfillment']['speech'];

But it just comes up blank.. How am I supposed to know if api.ai is responding and how would I display the response, thanks again @christophrumpel !!
